DEBUG 2017-05-12 11:43:00,576: 	 Executing command: 'submit'
DEBUG 2017-05-12 11:43:00,578: 	 Found CRAB cache file /afs/cern.ch/user/a/aevans/.crab3
DEBUG 2017-05-12 11:43:00,589: 	 Checking credentials
DEBUG 2017-05-12 11:43:00,603: 	 Found proxy file /tmp/x509up_u70979
DEBUG 2017-05-12 11:43:00,603: 	 Getting proxy life time left
DEBUG 2017-05-12 11:43:01,625: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u70979 -timeleft
 output : 82053

 error: 
 retcode : 0
DEBUG 2017-05-12 11:43:02,782: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u70979 -actimeleft
 output : 82052

 error: 
 retcode : 0
DEBUG 2017-05-12 11:43:02,782: 	 Proxy valid for 22:47:33 hours
INFO 2017-05-12 11:43:02,783: 	 Will use CRAB configuration file crabConfig_WR_M-2400_ToLNu_M-160_RECO_MuMuJJ.py
DEBUG 2017-05-12 11:43:02,783: 	 Loading CRAB configuration file.
ERROR 2017-05-12 11:43:02,794: 	 Error while loading CRAB configuration:
name 'nInputFiles' is not defined
Please refer to https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3CommonErrors#Syntax_error_in_CRAB_configurati
See the ./crab.log file for more details.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2017-05-12 11:43:02,794: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 298, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 84, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Error while loading CRAB configuration:
name 'nInputFiles' is not defined
Please refer to https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3CommonErrors#Syntax_error_in_CRAB_configurati
See the ./crab.log file for more details.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
DEBUG 2017-05-12 11:43:03,709: 	 Executing command: 'submit'
DEBUG 2017-05-12 11:43:03,710: 	 Found CRAB cache file /afs/cern.ch/user/a/aevans/.crab3
DEBUG 2017-05-12 11:43:03,722: 	 Checking credentials
DEBUG 2017-05-12 11:43:03,722: 	 Found proxy file /tmp/x509up_u70979
DEBUG 2017-05-12 11:43:03,722: 	 Getting proxy life time left
DEBUG 2017-05-12 11:43:04,846: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u70979 -timeleft
 output : 82050

 error: 
 retcode : 0
DEBUG 2017-05-12 11:43:05,973: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u70979 -actimeleft
 output : 82049

 error: 
 retcode : 0
DEBUG 2017-05-12 11:43:05,974: 	 Proxy valid for 22:47:30 hours
INFO 2017-05-12 11:43:05,974: 	 Will use CRAB configuration file crabConfig_WR_M-2400_ToLNu_M-800_RECO_MuMuJJ.py
DEBUG 2017-05-12 11:43:05,974: 	 Loading CRAB configuration file.
ERROR 2017-05-12 11:43:05,989: 	 Error while loading CRAB configuration:
name 'nInputFiles' is not defined
Please refer to https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3CommonErrors#Syntax_error_in_CRAB_configurati
See the ./crab.log file for more details.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2017-05-12 11:43:05,989: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 298, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 84, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Error while loading CRAB configuration:
name 'nInputFiles' is not defined
Please refer to https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3CommonErrors#Syntax_error_in_CRAB_configurati
See the ./crab.log file for more details.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
DEBUG 2017-05-12 11:43:06,554: 	 Executing command: 'submit'
DEBUG 2017-05-12 11:43:06,554: 	 Found CRAB cache file /afs/cern.ch/user/a/aevans/.crab3
DEBUG 2017-05-12 11:43:06,566: 	 Checking credentials
DEBUG 2017-05-12 11:43:06,566: 	 Found proxy file /tmp/x509up_u70979
DEBUG 2017-05-12 11:43:06,566: 	 Getting proxy life time left
DEBUG 2017-05-12 11:43:07,376: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u70979 -timeleft
 output : 82047

 error: 
 retcode : 0
DEBUG 2017-05-12 11:43:08,191: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u70979 -actimeleft
 output : 82046

 error: 
 retcode : 0
DEBUG 2017-05-12 11:43:08,192: 	 Proxy valid for 22:47:27 hours
INFO 2017-05-12 11:43:08,192: 	 Will use CRAB configuration file crabConfig_WR_M-4000_ToLNu_M-1333_RECO_MuMuJJ.py
DEBUG 2017-05-12 11:43:08,192: 	 Loading CRAB configuration file.
ERROR 2017-05-12 11:43:08,199: 	 Error while loading CRAB configuration:
name 'nInputFiles' is not defined
Please refer to https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3CommonErrors#Syntax_error_in_CRAB_configurati
See the ./crab.log file for more details.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2017-05-12 11:43:08,199: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 298, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 84, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Error while loading CRAB configuration:
name 'nInputFiles' is not defined
Please refer to https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3CommonErrors#Syntax_error_in_CRAB_configurati
See the ./crab.log file for more details.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
DEBUG 2017-05-12 11:43:08,735: 	 Executing command: 'submit'
DEBUG 2017-05-12 11:43:08,736: 	 Found CRAB cache file /afs/cern.ch/user/a/aevans/.crab3
DEBUG 2017-05-12 11:43:08,743: 	 Checking credentials
DEBUG 2017-05-12 11:43:08,744: 	 Found proxy file /tmp/x509up_u70979
DEBUG 2017-05-12 11:43:08,744: 	 Getting proxy life time left
DEBUG 2017-05-12 11:43:09,555: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u70979 -timeleft
 output : 82045

 error: 
 retcode : 0
DEBUG 2017-05-12 11:43:10,476: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u70979 -actimeleft
 output : 82044

 error: 
 retcode : 0
DEBUG 2017-05-12 11:43:10,476: 	 Proxy valid for 22:47:25 hours
INFO 2017-05-12 11:43:10,476: 	 Will use CRAB configuration file crabConfig_WR_M-4000_ToLNu_M-267_RECO_MuMuJJ.py
DEBUG 2017-05-12 11:43:10,476: 	 Loading CRAB configuration file.
ERROR 2017-05-12 11:43:10,495: 	 Error while loading CRAB configuration:
name 'nInputFiles' is not defined
Please refer to https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3CommonErrors#Syntax_error_in_CRAB_configurati
See the ./crab.log file for more details.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2017-05-12 11:43:10,495: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 298, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 84, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Error while loading CRAB configuration:
name 'nInputFiles' is not defined
Please refer to https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3CommonErrors#Syntax_error_in_CRAB_configurati
See the ./crab.log file for more details.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
DEBUG 2017-05-12 11:45:15,711: 	 Executing command: 'submit'
DEBUG 2017-05-12 11:45:15,712: 	 Found CRAB cache file /afs/cern.ch/user/a/aevans/.crab3
DEBUG 2017-05-12 11:45:15,719: 	 Checking credentials
DEBUG 2017-05-12 11:45:15,720: 	 Found proxy file /tmp/x509up_u70979
DEBUG 2017-05-12 11:45:15,720: 	 Getting proxy life time left
DEBUG 2017-05-12 11:45:16,528: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u70979 -timeleft
 output : 81918

 error: 
 retcode : 0
DEBUG 2017-05-12 11:45:17,438: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u70979 -actimeleft
 output : 81917

 error: 
 retcode : 0
DEBUG 2017-05-12 11:45:17,438: 	 Proxy valid for 22:45:18 hours
INFO 2017-05-12 11:45:17,439: 	 Will use CRAB configuration file crabConfig_WR_M-2400_ToLNu_M-160_RECO_MuMuJJ.py
DEBUG 2017-05-12 11:45:17,439: 	 Loading CRAB configuration file.
ERROR 2017-05-12 11:45:17,443: 	 Error while loading CRAB configuration:
name 'nInputFiles' is not defined
Please refer to https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3CommonErrors#Syntax_error_in_CRAB_configurati
See the ./crab.log file for more details.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2017-05-12 11:45:17,443: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 298, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 84, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Error while loading CRAB configuration:
name 'nInputFiles' is not defined
Please refer to https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3CommonErrors#Syntax_error_in_CRAB_configurati
See the ./crab.log file for more details.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
DEBUG 2017-05-12 11:45:17,976: 	 Executing command: 'submit'
DEBUG 2017-05-12 11:45:17,976: 	 Found CRAB cache file /afs/cern.ch/user/a/aevans/.crab3
DEBUG 2017-05-12 11:45:17,982: 	 Checking credentials
DEBUG 2017-05-12 11:45:17,983: 	 Found proxy file /tmp/x509up_u70979
DEBUG 2017-05-12 11:45:17,983: 	 Getting proxy life time left
DEBUG 2017-05-12 11:45:18,989: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u70979 -timeleft
 output : 81916

 error: 
 retcode : 0
DEBUG 2017-05-12 11:45:19,803: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u70979 -actimeleft
 output : 81915

 error: 
 retcode : 0
DEBUG 2017-05-12 11:45:19,803: 	 Proxy valid for 22:45:16 hours
INFO 2017-05-12 11:45:19,803: 	 Will use CRAB configuration file crabConfig_WR_M-2400_ToLNu_M-800_RECO_MuMuJJ.py
DEBUG 2017-05-12 11:45:19,804: 	 Loading CRAB configuration file.
ERROR 2017-05-12 11:45:19,834: 	 Error while loading CRAB configuration:
name 'nInputFiles' is not defined
Please refer to https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3CommonErrors#Syntax_error_in_CRAB_configurati
See the ./crab.log file for more details.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2017-05-12 11:45:19,834: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 298, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 84, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Error while loading CRAB configuration:
name 'nInputFiles' is not defined
Please refer to https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3CommonErrors#Syntax_error_in_CRAB_configurati
See the ./crab.log file for more details.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
DEBUG 2017-05-12 11:45:20,562: 	 Executing command: 'submit'
DEBUG 2017-05-12 11:45:20,562: 	 Found CRAB cache file /afs/cern.ch/user/a/aevans/.crab3
DEBUG 2017-05-12 11:45:20,570: 	 Checking credentials
DEBUG 2017-05-12 11:45:20,571: 	 Found proxy file /tmp/x509up_u70979
DEBUG 2017-05-12 11:45:20,571: 	 Getting proxy life time left
DEBUG 2017-05-12 11:45:21,591: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u70979 -timeleft
 output : 81913

 error: 
 retcode : 0
DEBUG 2017-05-12 11:45:22,299: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u70979 -actimeleft
 output : 81912

 error: 
 retcode : 0
DEBUG 2017-05-12 11:45:22,299: 	 Proxy valid for 22:45:13 hours
INFO 2017-05-12 11:45:22,299: 	 Will use CRAB configuration file crabConfig_WR_M-4000_ToLNu_M-1333_RECO_MuMuJJ.py
DEBUG 2017-05-12 11:45:22,299: 	 Loading CRAB configuration file.
ERROR 2017-05-12 11:45:22,302: 	 Error while loading CRAB configuration:
name 'nInputFiles' is not defined
Please refer to https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3CommonErrors#Syntax_error_in_CRAB_configurati
See the ./crab.log file for more details.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2017-05-12 11:45:22,302: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 298, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 84, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Error while loading CRAB configuration:
name 'nInputFiles' is not defined
Please refer to https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3CommonErrors#Syntax_error_in_CRAB_configurati
See the ./crab.log file for more details.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
DEBUG 2017-05-12 11:45:22,821: 	 Executing command: 'submit'
DEBUG 2017-05-12 11:45:22,821: 	 Found CRAB cache file /afs/cern.ch/user/a/aevans/.crab3
DEBUG 2017-05-12 11:45:22,828: 	 Checking credentials
DEBUG 2017-05-12 11:45:22,828: 	 Found proxy file /tmp/x509up_u70979
DEBUG 2017-05-12 11:45:22,828: 	 Getting proxy life time left
DEBUG 2017-05-12 11:45:23,534: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u70979 -timeleft
 output : 81911

 error: 
 retcode : 0
DEBUG 2017-05-12 11:45:24,243: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u70979 -actimeleft
 output : 81910

 error: 
 retcode : 0
DEBUG 2017-05-12 11:45:24,244: 	 Proxy valid for 22:45:11 hours
INFO 2017-05-12 11:45:24,244: 	 Will use CRAB configuration file crabConfig_WR_M-4000_ToLNu_M-267_RECO_MuMuJJ.py
DEBUG 2017-05-12 11:45:24,244: 	 Loading CRAB configuration file.
ERROR 2017-05-12 11:45:24,246: 	 Error while loading CRAB configuration:
name 'nInputFiles' is not defined
Please refer to https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3CommonErrors#Syntax_error_in_CRAB_configurati
See the ./crab.log file for more details.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2017-05-12 11:45:24,247: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 298, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 84, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Error while loading CRAB configuration:
name 'nInputFiles' is not defined
Please refer to https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3CommonErrors#Syntax_error_in_CRAB_configurati
See the ./crab.log file for more details.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
DEBUG 2017-05-12 11:57:31,770: 	 Executing command: 'submit'
DEBUG 2017-05-12 11:57:31,771: 	 Found CRAB cache file /afs/cern.ch/user/a/aevans/.crab3
DEBUG 2017-05-12 11:57:31,779: 	 Checking credentials
DEBUG 2017-05-12 11:57:31,780: 	 Found proxy file /tmp/x509up_u70979
DEBUG 2017-05-12 11:57:31,780: 	 Getting proxy life time left
DEBUG 2017-05-12 11:57:33,015: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u70979 -timeleft
 output : 81182

 error: 
 retcode : 0
DEBUG 2017-05-12 11:57:34,041: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u70979 -actimeleft
 output : 81181

 error: 
 retcode : 0
DEBUG 2017-05-12 11:57:34,042: 	 Proxy valid for 22:33:02 hours
INFO 2017-05-12 11:57:34,042: 	 Will use CRAB configuration file crabConfig_WR_M-2400_ToLNu_M-160_RECO_MuMuJJ.py
DEBUG 2017-05-12 11:57:34,042: 	 Loading CRAB configuration file.
DEBUG 2017-05-12 11:57:34,047: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2017-05-12 11:57:34,049: 	 Working area '/afs/cern.ch/user/a/aevans/public/wr/CMSSW_8_0_24_patch1/src/ExoAnalysis/cmsWRextension/scripts/crabDigiRawRecoJobs/crab_projects/crab_WR_M-2400_ToLNu_M-160_RECO_MuMuJJ' already exists 
Please change the requestName in the config file
ERROR 2017-05-12 11:57:34,049: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 302, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/lib/python2.7/site-packages/CRABClient/ClientUtilities.py", line 394, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/cern.ch/user/a/aevans/public/wr/CMSSW_8_0_24_patch1/src/ExoAnalysis/cmsWRextension/scripts/crabDigiRawRecoJobs/crab_projects/crab_WR_M-2400_ToLNu_M-160_RECO_MuMuJJ' already exists 
Please change the requestName in the config file
DEBUG 2017-05-12 11:57:34,551: 	 Executing command: 'submit'
DEBUG 2017-05-12 11:57:34,551: 	 Found CRAB cache file /afs/cern.ch/user/a/aevans/.crab3
DEBUG 2017-05-12 11:57:34,557: 	 Checking credentials
DEBUG 2017-05-12 11:57:34,558: 	 Found proxy file /tmp/x509up_u70979
DEBUG 2017-05-12 11:57:34,558: 	 Getting proxy life time left
DEBUG 2017-05-12 11:57:35,565: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u70979 -timeleft
 output : 81179

 error: 
 retcode : 0
DEBUG 2017-05-12 11:57:36,608: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u70979 -actimeleft
 output : 81178

 error: 
 retcode : 0
DEBUG 2017-05-12 11:57:36,608: 	 Proxy valid for 22:32:59 hours
INFO 2017-05-12 11:57:36,609: 	 Will use CRAB configuration file crabConfig_WR_M-2400_ToLNu_M-800_RECO_MuMuJJ.py
DEBUG 2017-05-12 11:57:36,609: 	 Loading CRAB configuration file.
DEBUG 2017-05-12 11:57:36,613: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2017-05-12 11:57:36,613: 	 Working area '/afs/cern.ch/user/a/aevans/public/wr/CMSSW_8_0_24_patch1/src/ExoAnalysis/cmsWRextension/scripts/crabDigiRawRecoJobs/crab_projects/crab_WR_M-2400_ToLNu_M-800_RECO_MuMuJJ' already exists 
Please change the requestName in the config file
ERROR 2017-05-12 11:57:36,613: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 302, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/lib/python2.7/site-packages/CRABClient/ClientUtilities.py", line 394, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/cern.ch/user/a/aevans/public/wr/CMSSW_8_0_24_patch1/src/ExoAnalysis/cmsWRextension/scripts/crabDigiRawRecoJobs/crab_projects/crab_WR_M-2400_ToLNu_M-800_RECO_MuMuJJ' already exists 
Please change the requestName in the config file
DEBUG 2017-05-12 11:57:37,391: 	 Executing command: 'submit'
DEBUG 2017-05-12 11:57:37,391: 	 Found CRAB cache file /afs/cern.ch/user/a/aevans/.crab3
DEBUG 2017-05-12 11:57:37,399: 	 Checking credentials
DEBUG 2017-05-12 11:57:37,400: 	 Found proxy file /tmp/x509up_u70979
DEBUG 2017-05-12 11:57:37,400: 	 Getting proxy life time left
DEBUG 2017-05-12 11:57:38,312: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u70979 -timeleft
 output : 81176

 error: 
 retcode : 0
DEBUG 2017-05-12 11:57:39,727: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u70979 -actimeleft
 output : 81175

 error: 
 retcode : 0
DEBUG 2017-05-12 11:57:39,727: 	 Proxy valid for 22:32:56 hours
INFO 2017-05-12 11:57:39,727: 	 Will use CRAB configuration file crabConfig_WR_M-4000_ToLNu_M-1333_RECO_MuMuJJ.py
DEBUG 2017-05-12 11:57:39,727: 	 Loading CRAB configuration file.
DEBUG 2017-05-12 11:57:39,752: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2017-05-12 11:57:39,753: 	 Working area '/afs/cern.ch/user/a/aevans/public/wr/CMSSW_8_0_24_patch1/src/ExoAnalysis/cmsWRextension/scripts/crabDigiRawRecoJobs/crab_projects/crab_WR_M-4000_ToLNu_M-1333_RECO_MuMuJJ' already exists 
Please change the requestName in the config file
ERROR 2017-05-12 11:57:39,753: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 302, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/lib/python2.7/site-packages/CRABClient/ClientUtilities.py", line 394, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/cern.ch/user/a/aevans/public/wr/CMSSW_8_0_24_patch1/src/ExoAnalysis/cmsWRextension/scripts/crabDigiRawRecoJobs/crab_projects/crab_WR_M-4000_ToLNu_M-1333_RECO_MuMuJJ' already exists 
Please change the requestName in the config file
DEBUG 2017-05-12 11:57:40,783: 	 Executing command: 'submit'
DEBUG 2017-05-12 11:57:40,783: 	 Found CRAB cache file /afs/cern.ch/user/a/aevans/.crab3
DEBUG 2017-05-12 11:57:40,797: 	 Checking credentials
DEBUG 2017-05-12 11:57:40,797: 	 Found proxy file /tmp/x509up_u70979
DEBUG 2017-05-12 11:57:40,797: 	 Getting proxy life time left
DEBUG 2017-05-12 11:57:42,052: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u70979 -timeleft
 output : 81172

 error: 
 retcode : 0
DEBUG 2017-05-12 11:57:43,171: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u70979 -actimeleft
 output : 81171

 error: 
 retcode : 0
DEBUG 2017-05-12 11:57:43,171: 	 Proxy valid for 22:32:52 hours
INFO 2017-05-12 11:57:43,171: 	 Will use CRAB configuration file crabConfig_WR_M-4000_ToLNu_M-267_RECO_MuMuJJ.py
DEBUG 2017-05-12 11:57:43,171: 	 Loading CRAB configuration file.
DEBUG 2017-05-12 11:57:43,187: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2017-05-12 11:57:43,188: 	 Working area '/afs/cern.ch/user/a/aevans/public/wr/CMSSW_8_0_24_patch1/src/ExoAnalysis/cmsWRextension/scripts/crabDigiRawRecoJobs/crab_projects/crab_WR_M-4000_ToLNu_M-267_RECO_MuMuJJ' already exists 
Please change the requestName in the config file
ERROR 2017-05-12 11:57:43,188: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 302, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1705.patch3/lib/python2.7/site-packages/CRABClient/ClientUtilities.py", line 394, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/cern.ch/user/a/aevans/public/wr/CMSSW_8_0_24_patch1/src/ExoAnalysis/cmsWRextension/scripts/crabDigiRawRecoJobs/crab_projects/crab_WR_M-4000_ToLNu_M-267_RECO_MuMuJJ' already exists 
Please change the requestName in the config file
